src/components/ListItem.jsx

```diff
@@ -38,7 +46,42 @@ export function ListItem({ name, itemId, dateLastPurchased }) {
 		});
 	};
 
+	//Function to assign string value to buyingUrgency and color value to colorUrgency
+	const getBuyingUrgency = () => {
+		//Returns the difference between the currentDate and dateNextPurchased
+		const daysUntilNextPurchase = getDaysBetweenDates(
+			currentDate,
+			dateNextPurchased.toDate(),
+		);
+
+		//Conditional statement to categorize if an item is:
+		// - inactive: (60 days have passed since the last purchase)
+		// - overdue: currentDate has passed the dateNextPurchased, but not yet inactive
+		// - not soon: (30 days or more until the next purchase)
+		// - kind of soon: (between 7 & 30 days until the next purchase)
+		// - soon: (7 days or fewer until the next purchase)
+		if (daysUntilNextPurchase >= 60) {
+			buyingUrgency = 'inactive';
+			colorUrgency = '#878E88';
+		} else if (currentDate > dateNextPurchased.toDate()) {
+			buyingUrgency = 'overdue';
+			colorUrgency = '#A30000';
+		} else if (daysUntilNextPurchase >= 30) {
+			buyingUrgency = 'not soon';
+			colorUrgency = '#004777';
```

Comment:
Not too concerned with styling, but you may want to reconsider your color schema. For example, my browser uses dark mode, so `not soon` is pretty hard to see.
<img width="233" alt="Screenshot 2023-02-17 at 10 54 24 AM" src="https://user-images.githubusercontent.com/78327387/219759520-eaebaefb-ac67-4776-b0f7-e7ac53bd8052.png">

Reply:
Great point! We meant for the colors to be temporary placeholders for styling, but I agree that we can definitely make it a color that works for both dark and light mode for the time being.